src/graphql/schema/collection/createCollectionQueryFieldEntries.ts

```diff
@@ -34,7 +34,7 @@ export default function createCollectionQueryFieldEntries (
             type: gqlConditionType,
           }],
         ],
-        getPaginatorInput: (headValue: mixed, args: { condition?: { [key: string]: mixed } }) =>
+        getPaginatorInput: (_headValue: mixed, args: { condition?: { [key: string]: mixed } }) =>
```

Comment:
Why do we have to rename variables like `headValue` to `_headValue`? I’m worried in the future if we end up using these variables the leading underscore won’t get removed.